products/sle/main.pm

```diff
@@ -585,6 +585,11 @@ sub load_inst_tests {
         loadtest 'installation/add_update_test_repo';
     }
     loadtest "installation/addon_products_sle";
+
+    if (get_var('CHECK_RELEASENOTES')) {
+        loadtest 'installation/check_releasenotes';
+    }
+
```

Comment:
We don't use blank lines to separate the other if-blocks before and after so please remove the blank lines here as well

Reply:
Then let's fix our tidy script :)